packages/upset/src/components/Columns/Degree.tsx

```diff
@@ -12,10 +12,8 @@ export const Degree: FC<Props> = ({ degree }) => {
   return (
     <g
       transform={translate(
-        dimensions.matrixColumn.width +
-        dimensions.bookmarkStar.gap +
-        dimensions.bookmarkStar.width +
-        dimensions.bookmarkStar.gap,
+        dimensions.degreeColumn.width / 2 +
```

Comment:
Why `+ -6` instead of `- 6`? And I think this is more readable as a single line. The comment can go above the line if you don't wanna inline it. As far as centering goes, I'm guessing the issue is that the character starts at the center line when you do `width/2`, so subtracting 6px is necessary to move it back to the left the correct amount. Ideally this fixed amount would be based on the font size somehow, but I'm not sure that's possible and it'd be hacky if so. I think this solution works fine; if we change the font size for this column we'll notice the decentering.

Reply:
This is just personal preference to know that the change is a negative x direction. Force of habit from physics. Works the same either way

Reply:
Ok, I think that's fine to leave to personal preference